feat: add support for validating Arazzo 1.1.0 (#2877)
Redocly/redocly-cli: docs/@v2/guides/lint-arazzo.md

Choose from the ready-made rulesets (`spec`, `minimal`, `recommended` or `recommended-strict`), or go one better and configure the rules that suit your use case.
There's a full [list of built-in rules for Arazzo](../rules/built-in-rules.md#arazzo-rules) to refer to.

Add the rules to `redocly.yaml`.
For Arazzo description files, add rules in their own version-specific configuration section:
- `arazzo1Rules` for Arazzo 1.0.x documents

> - `arazzo1_1Rules` for Arazzo 1.1.x documents

Reviewer: I'd remove this section at all and cleaned up all examples referencing the `arazzo1Rules` field.

Reviewer: Just use regular rules.

Choose from the ready-made rulesets (`spec`, `minimal`, `recommended` or `recommended-strict`), or go one better and configure the rules that suit your use case.
There's a full [list of built-in rules for Arazzo](../rules/built-in-rules.md#arazzo-rules) to refer to.

> Add the rules to `redocly.yaml`, but for Arazzo specifications, the rules go in their own version-specific configuration section: `arazzo1Rules` for Arazzo 1.0.x documents, and `arazzo1_1Rules` for Arazzo 1.1.x documents.

Reviewer 2: ```suggestion
Add the rules to `redocly.yaml`.
For Arazzo description files, add rules in their own version-specific configuration section:
- `arazzo1Rules` for Arazzo 1.0.x documents
- `arazzo1_1Rules` for Arazzo 1.1.x documents
```
